msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md

```diff
@@ -0,0 +1,236 @@
+|**NotificationSignal** |Meeting signals will be delivered using the following existing conversation notification API (for user-bot chat). This API allows developers to signal based on end-user action to show-case an in-meeting dialog bubble.|**POST** _**/v3/conversations/{conversationId}/activities**_|Microsoft Bot Framework SDK|
+|**GetUserContext**| Get contextual information to display relevant content in a Teams tab. |_**microsoftTeams.getContext( ( ) => {  /*...*/ } )**_|Microsoft Teams client SDK|
+
+### GetParticipant API
+
+#### Request
+
+```http
+GET /v1/meetings/{meetingId}/participants/{participantId}?tenantId={tenantId}
+```
+
+#### Query parameters
+
+**meetingId**. The meeting identifier is required.  
+**participantId**. The participant identifier is required.  
+**tenantId**. Tenant id of the participant. Required for tenant user.
+
+#### Response Payload
+<!-- markdownlint-disable MD036 -->
+
+**Example 1**
+
+```json
+{
+    "meetingRole":"<meeting role>",
+    "conversation":
+        {
+            "id": "<conversation id>"
+        }
+}
+```
+
+**meetingRole** can be *organizer*, *presenter*, or *attendee*.
+
+**Example 2**
+
+```json
+{
+    "meetingRole": "Attendee",
+    "conversation": {
+        "isGroup": true,
+        "id": "sample" }
+}
+```
+
+#### Response Codes
+
+**200**: participant information successfully retrieved  
+**401**: invalid token  
+**403**: the app is not allowed to get participant information. There can be many reasons: app disabled by tenant admin, blocked during live site mitigation, etc.    
+**404**: the meeting doesn't exist or participant can’t be found    
+
+<!-- markdownlint-disable MD024 -->
+### NotificationSignal API
+
+#### Request
+
+```http
+POST /v3/conversations/{conversationId}/activities
+```
+
+#### Query parameters
+
+**conversationId**: The conversation identifier. Required
+
+#### Request Payload
+
+**Example 1**
+
+```json
+{
+"channelData": {
+    "notification": {
+        "alertInMeeting": true,
+
+        "externalResourceUrl: "https://teams.microsoft.com/l/bubble/APP_ID?url=<TaskInfo.url>&height=<TaskInfo.height>      &width=<TaskInfo.width> &title=<TaskInfo.title> ”
+                      }
+                 }
+            }
+```
+
+**Example 2**
+
+```nodejs
+const replyActivity = MessageFactory.text('Hi'); // this could be an adaptive card instead
+        replyActivity.channelData = {
+            notification: {
+                alertInMeeting: true,
+                externalResourceUrl: 'https://teams.microsoft.com/l/bubble/APP_ID?url=<TaskInfo.url> &height=<TaskInfo.height>      &width=<TaskInfo.width> &title=<TaskInfo.title>’
+            }
+        };
+        await context.sendActivity(replyActivity);
+```
+
+**Example 3**
+
+```csharp
+var replyActivity = MessageFactory.text('Hi'); // this could be an adaptive card insteadreplyActivity.ChannelData = new TeamsChannelData{
```

Comment:
Um, this doesn't look like correct C#
- single quoted strings
- NotificationInfo in BF SDK hasn't been updated yet, has it? (https://github.com/microsoft/botbuilder-dotnet/blob/3a0dcfee003a7fcd0271c6336ab2c166f9a74313/libraries/Microsoft.Bot.Schema/Teams/Generated/NotificationInfo.cs)
- method casing

Reply:
✔ Resolution Pending